ui/imports/shared/popups/keycard/states/SeedPhrase.qml

```diff
@@ -0,0 +1,125 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+import QtGraphicalEffects 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+
+Item {
+    id: root
+
+    property var sharedKeycardModule
+    property bool hideSeed: true
+
+    signal seedPhraseRevealed()
+
+    QtObject {
+        id: d
+
+        readonly property var seedPhrase: root.sharedKeycardModule.getMnemonic().split(" ")
+    }
+
+    ColumnLayout {
+        anchors.fill: parent
+        anchors.topMargin: Style.current.xlPadding
+        anchors.bottomMargin: Style.current.halfPadding
+        anchors.leftMargin: Style.current.xlPadding
+        anchors.rightMargin: Style.current.xlPadding
+        spacing: Style.current.padding
+        clip: true
```

Comment:
why `clip`? unnecessary clipping badly hurts performance

Reply:
the same reason as above